docker/settings_import.py

```diff
@@ -51,12 +44,7 @@
 
 # Read the preferred time zone from $TZ, use system locale or
 # set to 'America/New_York' if neither are set
-if getenv("TZ"):
-    if "/" in getenv("TZ"):
-        TIME_ZONE = getenv("TZ")
-    else:
-        TIME_ZONE = "America/New_York"
-elif getenv("TZ"):
+if getenv("TZ") and getenv("TZ") in pytz.all_timezones:
```

Comment:
I don't think this error should be silently eaten - if someone chooses a wrong timezone, the application should error.

Reply:
Happy to implement but didn't want to change the existing behaviour. 

Would you prefer warn or fatal ?

Reply:
Fatal sounds good to me